src/Button.svelte

```diff
@@ -116,7 +123,7 @@
   {title}
   {disabled}
   {style}
-  on:click|stopPropagation
+  on:click
```

Comment:
I removed the `stopPropagation` parameter here, since I think we should handle this in the component that uses the Button, so each component can decide if it wants to stop propagation

Reply:
Okay I think I have to rethink this one.. 🤔

Reply:
Ok in the end i reverted this, so resolved